packages/controller/lib/setup/setupBackup.js

```diff
@@ -629,6 +629,9 @@ class BackupRestore {
     }
 
     _copyBackupedFiles(backupDir) {
+        if (!fs.existsSync(backupDir)) {
+            return;
```

Comment:
is it really okay or should we better log or abort?

Reply:
I also had the logging idea like "no files to restore". WIll add that